app/models/ncr/work_order.rb

```diff
@@ -193,19 +193,55 @@ def system_approver_emails
     end
 
     def self.ba61_tier1_budget_mailbox
-      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com'
+      users = User.with_role('BA61_tier1_budget_approver').where(client_slug: 'ncr')
+      if !users or users.length == 0
+        user = User.with_email_role_slug!(
+          ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
Maybe worth moving into `db/seeds.rb`?

http://guides.rubyonrails.org/active_record_migrations.html#migrations-and-seed-data